CCS/Guadaloop_lib/guadaloop/lib/sensors/temperature_sensor.h

```diff
@@ -25,6 +25,9 @@ typedef int16_t temp_t;
  */
 typedef enum temperature_sensor_location {
     //TODO: add locations here
```

Comment:
maybe make these locations more specific (i.e. FRONT_LEFT, etc.) when we find out where the sensors will be placed. this should be fine for now